client/src/rr/dnssec/signer.rs

```diff
@@ -499,21 +499,6 @@ impl Signer {
     }
 }
 
-#[cfg(any(feature = "openssl", feature = "ring"))]
```

Comment:
ditto

Reply:
Again, it turns out that there are easy ways to do the same thing without this. Also, in normal (non-test) usage signer doesn't need to verify its own signatures.